pkg/pxc/app/volumes.go

```diff
@@ -0,0 +1,48 @@
+package app
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	api "github.com/Percona-Lab/percona-xtradb-cluster-operator/pkg/apis/pxc/v1alpha1"
+)
+
+func GetConfigVolumes(cr *api.PerconaXtraDBCluster, cvName string) corev1.Volume {
+	vol1 := corev1.Volume{
+		Name: "config-volume",
+	}
+
+	vol1.ConfigMap = &corev1.ConfigMapVolumeSource{}
+	vol1.ConfigMap.Name = cvName
+	t := true
+	vol1.ConfigMap.Optional = &t
+	return vol1
+}
+
+func GetDataVolumes(cr *api.PerconaXtraDBCluster, sfs api.StatefulApp, dataVolumeName string) (bool, []corev1.PersistentVolumeClaim, corev1.VolumeSource) {
```

Comment:
1. `cr *api.PerconaXtraDBCluster` - I suppose here is should be `api.VolumeSpec`. Now this function doesn't work for `Proxy`
2. Argument `sfs api.StatefulApp` is never used in function